src/tools/gophish_complete.py

```diff
@@ -6,23 +6,22 @@
 flag is provided with a CAMPAIGN_ID that campaign will be completed.
 
 Usage:
-  gophish-complete (--summary | --complete | --auto=CAMPAIGN_ID) [--log-level=LEVEL] SERVER API_KEY
+  gophish-complete (--auto=CAMPAIGN_ID | --complete | --summary ) [--log-level=LEVEL] SERVER API_KEY
   gophish-complete (-h | --help)
   gophish-complete --version
 
-
 Options:
-  SERVER 	--> Full URL to GoPhish server
-  API_KEY 	--> API Access Key
-  -c --complete   Completes Campaign and shows summary
-  -s --summary    Displays Campaign Summary
-  -h --help      Show this screen.
-  --version      Show version.
+  API_KEY                   API Access Key.
+  SERVER                    Full URL to GoPhish server.
+  -a --auto=CAMPAIGN_ID     Complete a specific campaign identified by CAMPAIGN_ID.
+  -c --complete             Complete Campaign and show summary.
```

Comment:
It's not clear from these descriptions what the difference is between the `--auto` and `--complete`.  

If the only difference is that one takes the campaign ID and the other prompts the user for one, then I'd suggest doing the following (in a separate PR since it will require some logic changes to the code; feel free to log an issue for this if you don't plan to make that change soon- I do that if I'm worried I may forget about a change I want to make):
* Get rid of the `--auto` option
* Support an optional `CAMPAIGN_ID` for both `--complete` and `--summary`
So the docopt line would look something like:
```
gophish-complete (--complete [CAMPAIGN_ID] | --summary [CAMPAIGN_ID]) [--log-level=LEVEL] SERVER API_KEY
```

Reply:
Great point. I will make an issue and get this worked out in a separate PR.

Reply:
I have created issue #2 for this suggestion with the plan to make these changes this week.  As part of this PR, I added a Note in the docopt and changed the comments for these options:

```
NOTE:
  * If the auto flag is not provided, all assessment campaigns will be listed to select from.
```

```
  -a --auto=CAMPAIGN_ID     Complete a specific campaign identified by CAMPAIGN_ID.
  -c --complete             Select and complete a campaign.```